This fixes inexact calculations on POWER, see https://github.com/xianyi/OpenBLAS/issues/2923
From https://github.com/xianyi/OpenBLAS/pull/3062

Backported by Alexander Grund (TU Dresden)

---
 param.h | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/param.h b/param.h
index f3ddde6..35f579b 100644
--- a/param.h
+++ b/param.h
@@ -2397,6 +2397,9 @@ USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #define GEMM_DEFAULT_OFFSET_B 65536
 #define GEMM_DEFAULT_ALIGN 0x0ffffUL
 
+#define SWITCH_RATIO            16
+#define GEMM_PREFERED_SIZE      16
+
 #define SGEMM_DEFAULT_UNROLL_M 16
 #define SGEMM_DEFAULT_UNROLL_N 8
 #define DGEMM_DEFAULT_UNROLL_M 16
